imagepipeline-backends/imagepipeline-okhttp3/src/main/java/com/facebook/imagepipeline/backends/okhttp3/OkHttpNetworkFetcher.java

```diff
@@ -54,18 +54,26 @@ public OkHttpNetworkFetchState(
   private static final String TOTAL_TIME = "total_time";
   private static final String IMAGE_SIZE = "image_size";
 
-  private final OkHttpClient mOkHttpClient;
+  private final Call.Factory mCallFactory;
 
   private Executor mCancellationExecutor;
 
   /**
    * @param okHttpClient client to use
    */
   public OkHttpNetworkFetcher(OkHttpClient okHttpClient) {
-    mOkHttpClient = okHttpClient;
+    mCallFactory = okHttpClient;
     mCancellationExecutor = okHttpClient.dispatcher().executorService();
```

Comment:
Instead of setting both, you can call `this(okHttpClient, okHttpClient.dispatcher().executorService())`